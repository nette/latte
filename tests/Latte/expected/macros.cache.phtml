<?php
// prolog Nette\Latte\Macros\CoreMacros
list($_l, $_g) = Nette\Latte\Macros\CoreMacros::initRuntime($template, 'xxx')
;
// prolog Nette\Latte\Macros\CacheMacro
Nette\Latte\Macros\CacheMacro::initRuntime($template, $_g);

// prolog Nette\Latte\Macros\UIMacros

// snippets support
if (%A%}

//
// main template
//
?>
Noncached content

<?php if (Nette\Latte\Macros\CacheMacro::createCache($netteCacheStorage, 'xxx', $_g->caches, array($id, 'tags' => 'mytag'))) { ?>

<h1><?php echo Nette\Templating\Helpers::escapeHtml($template->upper($title), ENT_NOQUOTES) ?></h1>

<?php Nette\Latte\Macros\CoreMacros::includeTemplate('include.cache.latte', array('localvar' => 11) + $template->getParameters(), $_l->templates['xxx'])->render() ?>

<?php $_l->tmp = array_pop($_g->caches); if (!$_l->tmp instanceof stdClass) $_l->tmp->end(); }
