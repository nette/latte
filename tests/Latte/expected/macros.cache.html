Noncached content


<h1>HELLO</h1>

<p>Included file (11)</p>

	hello
